api/router.js

```diff
@@ -2,8 +2,10 @@ import express from 'express';
 const router = express.Router();
 import userRouter from './user/router.js';
 import groupRouter from './group/router.js';
+import loginRouter from './authentication/controller/controller.js'
 
 router.use('/user', userRouter);
 router.use('/group', groupRouter);
+router.post('/', loginRouter);
```

Comment:
Fix using of authentication middleware:

Steps:
1. change import of loginRouter inside api/router.js from './authentication/controller/controller.js' to import loginRouter from './authentication/router.js'
2. change router.post('/', loginRouter) to router.use('/login', loginRouter) inside api/router.js
3. go to /api/user/router and change const router = express.Router(); to const router = express.Router().use(authenticateToken);
4. go to /api/user/router and change const router = express.Router(); to const router = express.Router().use(authenticateToken);

*Note: For steps 3 and 4 import authenticateToken from lib/middleware/authenticate/index.js